pkgs/backend/sdk/vnscope/classify.py

```
@@ -147,7 +147,9 @@ def plot_heatmap_with_candlestick(
         # For integrated plotting, mpf.plot creates its own figure. To integrate heatmap, plot separately or use returnfig=True
         # Here, we'll let mpf.plot create its own figure for candlestick + volume, and plot heatmap separately if enabled
         if enable_heatmap:
-            fig_heatmap, ax_heatmap = plt.subplots(figsize=(15, 3))
+            fig_heatmap, ax_heatmap = plt.subplots(
```

Ruff (0.14.1)

Unpacked variable `fig_heatmap` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)